data-raw/DATASET.R

```diff
@@ -20,32 +20,56 @@ df_adverse_events <-
                size = n, replace = TRUE),
       grade = sample.int(5, n, replace = TRUE)
     )
-  ) %>%
+  ) |>
   dplyr::left_join(
     tibble::tibble(
       patient_id = paste("ID", 1:10),
       trt = sample(c("Drug A", "Drug B"), size = 10, replace = TRUE)
     ),
     by = "patient_id"
-  ) %>%
-  dplyr::arrange(patient_id, system_organ_class, adverse_event) %>%
+  ) |>
+  dplyr::arrange(patient_id, system_organ_class, adverse_event) |>
   dplyr::mutate(
     drug_attribution =
       sample(c("Unrelated", "Unlikely", "Possible", "Probably", "Definite"),
              dplyr::n(),
-             replace = TRUE) %>%
-      factor(levels = c("Unrelated", "Unlikely", "Possible", "Probably", "Definite"))
-  ) %>%
-  dplyr::select(patient_id, trt, dplyr::everything()) %>%
+             replace = TRUE) |>
+      factor(levels = c("Unrelated", "Unlikely", "Possible", "Probably", "Definite")),
+    any_complication = TRUE,
+    grade3_complication = grade >= 3
+  ) |>
+  dplyr::select(patient_id, trt, dplyr::everything()) |>
   labelled::set_variable_labels(
     patient_id = "Patient ID",
     trt = "Treatment Group",
     system_organ_class = "System Organ Class",
     adverse_event = "Adverse Event",
     grade = "Grade",
-    drug_attribution = "Drug Attribution"
+    drug_attribution = "Drug Attribution",
+    any_complication = "Any Grade Complication",
+    grade3_complication = "Grade 3+ Complication"
   )
 
-usethis::use_data(df_adverse_events, overwrite = TRUE)
+df_patient_characteristics <-
+  tibble::tibble(
+    patient_id = paste("ID", 1:100),
+    trt = sample(c("Drug A", "Drug B"), size = 100, replace = TRUE),
+    age = rnorm(100, 50, 10)
```

Comment:
Suggesting whole numbers on age.

```suggestion
    age = floor(rnorm(100, 50, 10))
```